llm/readme.md

```diff
@@ -170,30 +170,14 @@ in the emulator after running `yarn dev:up`.
 # had a number of problems with authorization
 GOOGLE_APPLICATION_CREDENTIALS=/path/to/application_default_credentials.json \
   firebase deploy --only functions:maple-llm --debug
-
-# Hit the function in production
-curl \
-  -X POST \
-  -H "Content-Type: application/json" \
-  -H "Authorization: Bearer $(gcloud auth print-identity-token)" \
-  -d '{"bill_id": "1234","bill_title": "A title","bill_text": "Some bill text"}' \
-  https://httpsflaskexample-ke6znoupgq-uc.a.run.app/summary
 ```
 
 ## Future work
 
-Currently, we are just using the built-in Flask server. We should switch to a
-production WSGI server, like `gunicorn`.
-
 The local emulator installation process is quite cumbersome and ideally the
 virtual environment was built during container instantiation instead of from
 within the Docker container (i.e. the "Deploying locally" docs above).
 
-The current API is a little wonky because we take `bill_id` **and** `bill_text`.
```

Comment:
IIUC you are correct this is no longer true... but I don't recall when we actually fixed it, so just want to double check with you that it is so.